puppet/chassis.rb

```diff
@@ -322,6 +322,8 @@ def self.update_extensions(config)
 			puts "\033[A\e[32mChecking for Chassis extension updates...\e[0m"
 			extensions = Dir.glob(@@extension_dir + '/*').map { |directory| File.basename( directory ) }
 			updates = self.updates_check(extensions, @@extension_dir)
+			# Remove the _global folder as it's a synced folder.
+			updates.delete('_global')
```

Comment:
Presumably `.delete` means remove from the array? Surprised it's not `.delete!()` or similar given it's modifying the array.

Reply:
Yep, checked: https://ruby-doc.org/core-2.5.0/Array.html#method-i-delete - weird, seems inconsistent for Ruby!